molecule/default/converge.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Converge
+  hosts: all
+  roles:
+    - role: caddy-ansible
+  tasks:
+    - name: Ensure ss is installed (for testinfra)
+      yum:
```

Comment:
Can we use `package` and remove the condition?

Reply:
I'm not sure that is possible because the package is named differently (it's iproute2 on Debian-based distros & also on Arch linux). I checked the docs and couldn't find a way to handle this without setting a variable to the name of the package & I'm not sure how it can easily be set depending on the OS in this context without using a conditional `set_fact` (which doesn't feel much of an improvement on the current solution). Do you know a nicer way to handle this?

Guess I should at least ensure it is installed for the other distros for consistency though.

Reply:
Ok, thats fine. You can leave it.

Reply:
I had already added it for the other distros before I saw this but can remove if you prefer.

Reply:
If you already added it, then that's also ok :) I just meant, that you can ignore my comment about `package`